tests/basic_integration/fns.sh

```
@@ -0,0 +1,144 @@
+PLAINTEXT="1234,567890"
+ID=$(date +%s)
+
+if [[ "$ROOT_DIR" != "$(pwd)" ]]; then 
+  echo "This script must be run from the root"
+  exit 1
+fi
+
+# Environment variables
+RPC_URL="ws://localhost:8545"
+
+PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
```

Shellcheck

[warning] PRIVATE_KEY appears unused. Verify use (or export if used externally).

(SC2034)